.circleci/config.yml

```diff
@@ -1,20 +1,35 @@
 version: 2
 jobs:
-  prepare web dependencies cache:
+  prepare dependencies cache:
     docker:
       - image: node:8
     steps:
       - checkout
       - restore_cache:
-          key: v1-cms-hitech-apd-{{ checksum "web/package-lock.json" }}
+          key: cms-hitech-apd-npm-{{ checksum "web/package-lock.json" }}-{{ checksum "api/package-lock.json" }}
+      - run:
+          name: update npm
+          command: npm install -g npm@6
       - run:
-          name: install dependencies
+          name: install web dependencies
           working_directory: ~/project/web
-          command: npm install
+          command: npm ci
+      - run:
+          name: install api dependencies
+          working_directory: ~/project/api
+          command: npm ci
+      - save_cache:
+          key: cms-hitech-apd-npm-{{ checksum "web/package-lock.json" }}-{{ checksum "api/package-lock.json" }}
+          paths:
+            - /root/.npm
```

Comment:
This saves off the npm cache.  `npm ci` deletes `node_modules` if it already exists and populates it fresh from `package-lock.json`.  We can still cache off `node_modules` within a build run (so downstream steps can use it), but it's useless between runs.  Instead, we can save the npm cache to speed up future runs of `npm ci`.